### PR TITLE
Add chat links

### PR DESCRIPTION
**Pull request by andrewda**
This adds chat links to each organization using the chattie npm
module.

Closes https://github.com/coala/gci-leaders/issues/53

### CODE REVIEW

File: lib/scrape.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 const GITHUB_API_BASE = 'https://api.github.com'
 
 const MIN_SEARCH_SCORE = 10
 
+const CHAT_IMAGES = {
+  GITTER: 'static/images/logos/gitter.png',
+  SLACK: 'static/images/logos/slack.png',
+  ZULIP: 'static/images/logos/zulip.png',
+  TELEGRAM: 'static/images/logos/telegram.png',
+  IRC: 'static/images/logos/chat.png',
```
Comment (jayvdb) [inline lib/scrape.js]: maybe go with https://1.bp.blogspot.com/-600IhXfEKSA/Tv3jkZ3zAZI/AAAAAAAAA8E/5X7gdFm9Za0/s1600/1.png ?

**Comment by jayvdb** [comment]
The apertium irc link on https://deploy-preview-61--gci-leaders.netlify.com/ doesnt include the channel

**Review by Monal5031 — APPROVED** [review]
IMO the chats should be on next line of `Tasks Completed` but it isn't much difference though.

File: lib/scrape.js [hunk 2]
```diff
@@ -84,13 +95,19 @@ async function fetchOrgsWithData() {
...
   const orgGitHub = await Promise.all(fetchingGitHub)
+  const orgChats = await Promise.all(fetchingChat)
 
   return orgs.map((org, index) =>
     Object.assign(org, {
       leaders: orgLeaders[index],
       github: orgGitHub[index],
+      chat: {
+        url: orgChats[index].url,
+        image: CHAT_IMAGES[chattie.CHAT[orgChats[index].type]],
```
Comment (blazeu) [inline lib/scrape.js]: I'd prefer the chat platform type to be included also, to give a bit of clarity in the JSON.

**Comment by blazeu** [comment]
ack https://github.com/coala/gci-leaders/pull/61/commits/a2ac6cb791e1b0eb3ce701cc18b8910f5f7b3e4c

**Review by blazeu — APPROVED** [review]

**Review by jayvdb — APPROVED** [review]

**Pull request merged**